internal/cmd/login.go

```diff
@@ -507,32 +498,37 @@ func promptServer() (string, error) {
 		return "", err
 	}
 
-	hosts := config.HostNames()
+	servers := config.Hosts
 
-	if len(hosts) == 0 {
-		return promptNewHost()
+	if len(servers) == 0 {
+		return promptNewServer()
 	}
 
-	return promptExistingHosts(hosts)
+	return promptExistingServers(servers)
 }
 
-func promptNewHost() (string, error) {
-	var host string
-	err := survey.AskOne(&survey.Input{Message: "Host:"}, &host, survey.WithStdio(os.Stdin, os.Stderr, os.Stderr), survey.WithValidator(survey.Required))
+func promptNewServer() (string, error) {
+	var server string
+	err := survey.AskOne(&survey.Input{Message: "Server:"}, &server, survey.WithStdio(os.Stdin, os.Stderr, os.Stderr), survey.WithValidator(survey.Required))
 	if err != nil {
 		return "", err
 	}
 
-	return host, nil
+	return server, nil
 }
 
-func promptExistingHosts(hosts []string) (string, error) {
-	const defaultOption string = "Connect to a different host"
-	hosts = append(hosts, defaultOption)
+func promptExistingServers(servers []ClientHostConfig) (string, error) {
```

Comment:
function name isn't reflective of the options since adding a new server is an option so "Existing" is misleading

Reply:
perhaps something like `promptForServerFromList`?